spec/features/guest_can_view_total_amount_of_items_in_their_cart_on_the_navigation_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.feature "Guest sees total amount of items in their cart on the navigation" do
+  scenario "guest can see total items on navbar" do
+    album = FactoryGirl.create(:album)
```

Comment:
Go ahead and get rid of the variable assignment on line 5. All you need is `FactoryGirl.create(:album)`